src/analyze/accumulatedtracedata.cpp

```diff
@@ -195,7 +239,16 @@ bool AccumulatedTraceData::read(istream& in, const ParsePass pass, bool isRepars
     // allocations, i.e. when a deallocation follows with the same data
     uint64_t lastAllocationPtr = 0;
 
+    auto const uncompressedCount = in.component<byte_counter>(0);
+    auto const compressedCount   = in.component<byte_counter>(2);
+
     while (timeStamp < filterParameters.maxTime && reader.getLine(in)) {
+        parsingState.pass = pass;
```

Comment:
pass and reparsing only need to be set once above the loop

timestamp only needs to be updated when it changes below in the 'c' conditional. actually, I would even go as far as updating the bytes-read counter there too